graylog2-web-interface/src/stores/indices/IndexSetsStore.ts

```diff
@@ -278,6 +279,10 @@ export const IndexSetsStore = singletonStore(
 
     _errorMessage(error) {
       try {
+        if (isArray(error.additional.body)) {
+          return error.additional.body.map(({ message, path }) => `${path ?? ''} ${message}`).join(' ; ');
```

Comment:
Currently the generated error message looks like this:
`Creating index set 'test' failed with status: IndexSetsResource.save.arg0.shards must be greater than or equal to 1 ; IndexSetsResource.save.arg0.shards must be greater than or equal to 1`

Maybe we can use `.` for the separator and add one at the end?